test/SourceKit/CursorInfo/injected_vfs.swift

```diff
@@ -10,7 +10,7 @@ func foo(
 
 // CHECK-CMODULE: key.kind: source.lang.swift.ref.struct
 // CHECK-CMODULE: key.name: "StructDefinedInCModule"
-// CHECK-CMODULE: key.filepath: "/CModule/CModule.h"
+// CHECK-CMODULE: key.filepath: "{{.*}}/CModule{{/|\\\\}}CModule.h"
```

Comment:
Why is one slash here always a forward slash, but the other can be a backslash?

Reply:
Welcome to Windows™️! :D

(Yes, one of them was a `/` even on Windows, so I didn't replace it. Adding the `{{.*}}` accounts for the new path relative to `%t`, while the `{{/|\\\\}}` accounts for an escaped `\` in Windows.